locale/fr.json

```diff
@@ -0,0 +1,2122 @@
+{
+    "login": {
+        "copy": {
+            "translation": "Si vous avez déjà un compte human-connection, connectez-vous ici.",
+            "notes": ""
+        },
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
+            "notes": ""
+        },
+        "password": {
+            "translation": "Votre mot de passe",
+            "notes": ""
+        },
+        "moreInfo": {
+            "translation": "Qu'est-ce que Human Connection?",
+            "notes": ""
+        },
+        "hello": {
+            "translation": "Bonjour",
+            "notes": ""
+        },
+        "forgotPassword": {
+            "translation": "Mot de passe oublié?",
+            "notes": ""
+        },
+        "no-account": {
+            "translation": "Vous n'avez pas de compte?",
+            "notes": ""
+        },
+        "register": {
+            "translation": "S'inscrire",
+            "notes": ""
+        },
+        "moreInfoURL": {
+            "translation": "https:\/\/human-connection.org\/fr\/",
+            "notes": ""
+        },
+        "moreInfoHint": {
+            "translation": "à la page de présentation",
+            "notes": ""
+        },
+        "success": {
+            "translation": "Vous êtes connecté!",
+            "notes": ""
+        },
+        "failure": {
+            "translation": "Adresse e-mail ou mot de passe incorrect.",
```

Comment:
> [<img alt="Elweyn" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Elweyn) **Authored by [Elweyn](https://github.com/Elweyn)**
_<time datetime="2019-11-18T12:25:16Z" title="Monday, November 18th 2019, 1:25:16 pm +01:00">Nov 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,2122 @@
+{
+    "login": {
+        "copy": {
+            "translation": "Si vous avez déjà un compte human-connection, connectez-vous ici.",
+            "notes": ""
+        },
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
+            "notes": ""
+        },
+        "password": {
+            "translation": "Votre mot de passe",
+            "notes": ""
+        },
+        "moreInfo": {
+            "translation": "Qu'est-ce que Human Connection?",
+            "notes": ""
+        },
+        "hello": {
+            "translation": "Bonjour",
+            "notes": ""
+        },
+        "forgotPassword": {
+            "translation": "Mot de passe oublié?",
+            "notes": ""
+        },
+        "no-account": {
+            "translation": "Vous n'avez pas de compte?",
+            "notes": ""
+        },
+        "register": {
+            "translation": "S'inscrire",
+            "notes": ""
+        },
+        "moreInfoURL": {
+            "translation": "https:\/\/human-connection.org\/fr\/",
+            "notes": ""
+        },
+        "moreInfoHint": {
+            "translation": "à la page de présentation",
+            "notes": ""
+        },
+        "success": {
+            "translation": "Vous êtes connecté!",
+            "notes": ""
+        },
+        "failure": {
+            "translation": "Adresse e-mail ou mot de passe incorrect.",
```


Changed 'e-mail' to 'mail' on localise